open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/data-engine-connectors/data-engine-proxy-connector/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>data-engine-connectors</artifactId>
+        <groupId>org.odpi.egeria</groupId>
+        <version>1.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Data Engine Proxy Connector</name>
+    <description>
+        A generic proxy connector for plugging in data engine connectors to sync data engine metadata.
+    </description>
+
+    <artifactId>data-engine-proxy-connector</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>open-connector-framework</artifactId>
+            <version>${open-metadata.version}</version>
```

Comment:
WIth #22 integrated you can (should) omit the version tags in child poms.
If any NEW dependencies are added, an entry should be made in the dependencyManagement section of the top level pom, with a property also from top level pom